iOS/Flows/Conversation/Cells/MessageCell.swift

```diff
@@ -119,12 +123,25 @@ class MessageCell: UICollectionViewCell {
 
         self.footerView.alpha = messageLayoutAttributes.detailAlpha
 
+        // Hide the emotions view if the cell is scrolled out of focus.
+        if messageLayoutAttributes.detailAlpha < 0.5 && self.content.areEmotionsShown {
+            self.content.setEmotions(areShown: false, animated: true)
+        }
```

Comment:
After using it a bit, I think it's better to auto hide the emotions when you scroll away from a message.